tests/messaging.test.js

```diff
@@ -0,0 +1,45 @@
+import { Client, ApiController, FileWrapperi, MessagingExceptionError, MessageRequest } from '../src';
+
+let controller;
+
+beforeEach(() => {
+    const client = new Client({
+        basicAuthUserName: process.env.BANDWIDTH_USERNAME,
+        basicAuthPassword: process.env.BANDWIDTH_PASSWORD
+    });
+
+    controller = new ApiController(client);
+});
+
+describe('messaging', () => {
+    
+    it('should create message with proper values', async () => {
+        const accountId = process.env.BANDWIDTH_ACCOUNT_ID;
+        const body = {
+            applicationId: process.env.MESSAGING_APPLICATION_ID,
+            to: [process.env.PHONE_NUMBER_INBOUND],
+            from: process.env.PHONE_NUMBER_OUTBOUND,
+            text: 'TS messaging test'
+        };
+
+        const createMessageResponse = await controller.createMessage(accountId, body);
+        expect(createMessageResponse.result.applicationId).toBeDefined();
+    });
+    
+
+    it('should throw error on invalid values', async () => {
```

Comment:
I'd recommend a single incorrect value in the request to make the test more explicit. Maybe this becomes an "invalid application id" test.

Reply:
Good catch, actually the other SDK tests have an invalid "to" number so I'll just change it to that for consistency. Updated